Bump ODS to v11.2.0

pascalwengerter opened the pull request:
## Description
Lots of small enhancements incoming. Took the freedom to cleanup some dependency stuff and fix typos as well.

ownclouders commented:
Results for <strong>oC10SharingPublicExpireAndRoles</strong> https://drone.owncloud.com/owncloud/web/20173/37/1
The following scenarios passed on retry:
- webUISharingPublicExpire/shareByPublicLinkExpiringLinks.feature:11

pascalwengerter commented:
Rebased to resolve lockfile conflicts, should fail in CI since we deleted the corresponding ODS release and will re-release it soon with a fix. Restarting CI then

kulmann reviewed (approved):
LGTM 👍
